### DISCUSSION TITLE
Login screen warning in i3wm

### DISCUSSION THREAD

**Discussion by panigrc** [rustdesk/rustdesk#1150] [Q&A]
Hi, I am using Ubuntu server 22.04 with i3wm. I've installed the Rustdesk using the .deb package.

I am not using a login screen at all, I am logging in through the console and starting the x session (`startx`) manually.

The warning that I'm using wayland is displayed:
![image](https://user-images.githubusercontent.com/645895/182300927-ba36c09a-3ce9-4672-84ad-4001fc510695.png)

Also I can't see any quit/exit button and `Ctrl+q` or `Ctrl+w` isn't working.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/921

**Reply by panigrc** [reply]
Alright, I'll wait for the next release then, or I'll built it.
